MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -417,21 +417,10 @@ open class RouteStepProgress: NSObject {
     }
     
     /**
-     Returns all the intersection distances on the current step.
+     Returns an array of the calculated distances from the current intersection to the next intersection on the current step.
```

Comment:
Nit: this is a property, not a method, so it is _set to_ the array rather than returning the array.